src/js/api.js

```diff
@@ -91,8 +117,11 @@ export function getLatestSubmission(){
   return sendFetch('/submissions/latest')
 }
 
-export function getEditsByType(submission){
-  return sendFetch(`/submissions/${submission}/edits`)
+export function getEditsByType(submission, institutionId, period, params){
+  if(params) {
+    return sendFetch(`/institutions/${institutionId}/filings/${period}/submissions/${submission}/edits`, {noParse:1, params:params})
+  }
+  return sendFetch(`/submissions/${submission}/edits`, {params:params})
```

Comment:
The last line of this fn can just be `return sendFetch(`/submissions/${submission}/edits`)